### ISSUE TITLE
Use modules instead of alias to resolve dependencies

### ISSUE THREAD

**Issue by zephraph** [egoist/vbuild#20]
When you build out the webpack config you're using this...

```
webpackConfig.resolve.alias = {
      src: _.cwd(srcPath),
      components: _.cwd(srcPath, 'components'),
      store: _.cwd(srcPath, 'vuex', 'store.js'),
      css: _.cwd(srcPath, 'css'),
      views: _.cwd(srcPath, 'views')
    }
```

That assumes a lot about someone's directory structure. It's also notably missing `router`. Regardless, my advice would be to instead do the following.

```
webpackConfig.resolve.modules.push(_.cwd(srcPath)
```

In general, gives a more robust way of configuring imports without going through the dot slash hell. It makes it where anything in the src directory can be directly imported as opposed to only the things that are explicitly aliased.

**Comment by egoist** [comment]
I didn't get it, how would `webpackConfig.resolve.modules.push(_.cwd(srcPath)` help with dot hell? I still need to do something like `import '../../components/x.vue'`?

**Comment by zephraph** [comment]
The `modules` configuration option tells webpack where to look to resolve dependencies, right? 

Given
```
/src
  /components
    x.vue
```

where `src` is specified in the `modules` configuration, `x` can be imported by doing

```
import x from 'components/x.vue'; 
```

`alias` tells webpack that if it encounters a certain thing in require then look for something else. `modules` actually gives it new items in the resolution path.

**Comment by egoist** [comment]
OK, I get it now 🙂

**Comment by zephraph** [comment]
I can do a PR.

**Comment by zephraph** [comment]
Resolved by #21.
